samples/graph-activity-feed/csharp/ActivityFeedNotification/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,226 @@
+﻿
+@model TabActivityFeed.Model.TaskInfo
+@{
+    Layout = "";
+}
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
Index.cshtml and GroupChatNotification.cshtml have too much repeat code, please check if it can be combined or moved to partial views

Reply:
Both are different one is for static tab (index.cshtml) and other is for chat, I think i can cross verify both and let you know

Reply:
I still see lot of repeat code. Can we discuss this?

Reply:
Both view are different and having different methods to be called